src/Test/Perf/Runner/Program.cs

```diff
@@ -81,11 +81,11 @@ public static void Main(string[] args)
                 testInstances.AddRange(tests);
             }
 
-            var traceManager = TraceManagerFactory.GetTraceManager();
 
-            traceManager.Initialize();
```

Comment:
I am not sure what the implications of moving the `Initialize` from outside the loop to inside the loop